jvm-packages/xgboost4j-spark-gpu/src/main/scala/ml/dmlc/xgboost4j/scala/rapids/spark/GpuUtils.scala

```diff
@@ -31,16 +31,7 @@ private[spark] object GpuUtils {
 
   def extractBatchToHost(table: Table, types: Array[DataType]): Array[ColumnVector] = {
     // spark-rapids has shimmed the GpuColumnVector from 22.10
-    try {
-      val clazz = Utils.classForName("com.nvidia.spark.rapids.GpuColumnVectorUtils")
-      clazz.getDeclaredMethod("extractHostColumns", classOf[Table], classOf[Array[DataType]])
-        .invoke(null, table, types).asInstanceOf[Array[ColumnVector]]
-    } catch {
-      case _: ClassNotFoundException =>
-        // If it's older version, use the GpuColumnVector
-        GpuColumnVector.extractColumns(table, types).map(_.copyToHost())
-      case e: Throwable => throw e
-    }
+    GpuColumnVectorUtils.extractHostColumns(table, types)
```

Comment:
We can now remove the workaround introduced from #8240

Reply:
Please don't remove this workaround for this time, or else xgboost can't work well with older spark-rapids (before 22.10). Let's remove it for the next release.